skrub/datasets/_fetching.py

```diff
@@ -12,7 +12,8 @@ def fetch_employee_salaries(data_home=None):
     Description of the dataset:
         Annual salary information including gross pay and overtime pay for all
         active, permanent employees of Montgomery County, MD paid in calendar
-        year 2016. This information will be published annually each year.
+        year 2016. This information will be published annually each year. Size on disk:
```

Comment:
This part was removed in a different PR
```suggestion
        year 2016. Size on disk:
```